tests/foreman/ui/test_contenthost.py

```diff
@@ -46,7 +46,7 @@
 from robottelo.constants import VDC_SUBSCRIPTION_NAME
 from robottelo.constants import VIRT_WHO_HYPERVISOR_TYPES
 from robottelo.utils.issue_handlers import is_open
-from robottelo.virtwho_utils import create_fake_hypervisor_content
+from robottelo.utils.virtwho_utils import create_fake_hypervisor_content
```

Comment:
As we moving this virtwho_utils module under utils, so can we rename this to `virtwho` now as similar to installer/version modules? non-blocking :)

Reply:
I will leave it for Component owner @yanpliu to update it if she likes ! :)

Reply:
@jyejare If we rename module, I don't think it will have any impact on component, but better to confirm from @yanpliu